source/opt/pass.h

```diff
@@ -67,9 +67,14 @@ class Pass {
   // Add to |todo| all ids of functions called in |func|.
   void AddCalls(ir::Function* func, std::queue<uint32_t>* todo);
 
-  // 
+  // Applies |pfn| to every function in the call trees that are rooted at the
```

Comment:
I guess I forgot to ask for the description the first time around.
Must say what the return value means:
"Returns true if any call to the process function returns true.  By convention the process function should return true if it modified the module."